dev/Repeater/APITests/SelectionModelTests.cs

```diff
@@ -980,6 +980,118 @@ public void SelectRangeRegressionTest()
             });
         }
 
+        [TestMethod]
+        public void AlreadySelectedDoesNotRaiseEvent()
+        {
+            var testName = "Select(int32 index), single select";
+
+            RunOnUIThread.Execute(() =>
+            {
+                var list = Enumerable.Range(0, 10).ToList();
+
+                var selectionModel = new SelectionModel() {
+                    Source = list,
+                    SingleSelect = true
+                };
+
+                // Single select index
+                selectionModel.Select(0);
+                selectionModel.SelectionChanged += SelectionModel_SelectionChanged;
```

Comment:
nit: could make this describe what it does better by naming it ThrowIfRaised or something similar.

Reply:
Yeah, the naming was not great. Fixed.